src/Auth/ResetsPasswords.php

```diff
@@ -75,7 +75,7 @@ protected function rules()
         return [
             'token' => 'required',
             'email' => 'required|email',
-            'password' => 'required|confirmed|min:8',
+            'password' => array_merge(['required', 'confirmed'], config('statamic.users.password_validation', [])),
```

Comment:
If you go down the route of adding the default password rules in the config, it means you'll no longer need the `array_merge` here anymore (or in the other places you're doing it).

You'll probably also want to fallback to the default rules, in case the user doesn't add the option to their config file.

```php
'password' => config('statamic.users.password_validation', ['required', 'confirmed']),
```

Reply:
Same as in the first comment, as `confirmed` is not be needed everywhere (see MakeUser command) I see little benefit in it.

IMHO it is safer and cleaner to have the hardcoded rules and adding additional rules.

I would rather drop support for PHP < 7.4 and be able to use 😂 
``` php
['required', 'confirmed', ...config('statamic.users.password_validation')]
```

Reply:
Yeah, I see where you're coming from...